openapi.yaml

```diff
@@ -4958,6 +4958,62 @@ paths:
           $ref: '#/components/responses/HTTP401Error'
       tags:
         - Plugins
+  '/v2/control-planes/{controlPlaneId}/core-entities/plugins/{PluginId}#KonnectApplicationAuth':
```

Comment:
@mheap `KonnectApplicationAuth` plugin is also not supposed to be there, correct?
Since we removed this in [v2.4.1](https://github.com/Kong/terraform-provider-konnect/commit/ff564f8cada81f155aa9e2626ea83e70c5bf1823#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39L4958)

Reply:
That's correct. It looks like I forgot to push a generator change, I can take care of this in a couple of hours

Reply:
https://github.com/Kong/platform-api/pull/1133 is merged and resolves this